openqa-search-maintenance-core-jobs

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    cat << EOF
+Usage: $(basename "$0") -a SUSE:Maintenance:II:RR [-d days]
+
+The goal of this script is basically to fill the section
+"regression tests" in the log of QA test report,
+for a given update inside job groups:
+- Maintenance: Single Incidents / Core Incidents
+- Maintenance: Aggregated updates / Core Maintenance Updates
+and search for regression tests related with the update.
+Default search within the last 5 days for "Aggregated updates" builds
+
+Usage: eg
+  $(basename "$0") -a SUSE:Maintenance:34576:337335
+  $(basename "$0") -a SUSE:Maintenance:34577:337331
+  $(basename "$0") -a SUSE:Maintenance:34564:337279 -d 10
+  $(basename "$0") -a SUSE:Maintenance:34581:337626
+EOF
+}
+
+# Maintenance: Single Incidents / Core Incidents
+declare -A dict_group=(
+    ["15-SP1"]=233
+    ["15-SP2"]=306
+    ["15-SP3"]=367
+    ["15-SP4"]=439
+    ["15-SP5"]=490
+    ["15-SP6"]=546
+    ["12-SP3"]=106
+    ["12-SP5"]=282
+    ["15-SP4-TERADATA"]=521
+    ["12-SP3-TERADATA"]=191
+)
+
+readonly url_dashboard_qam='http://dashboard.qam.suse.de'
+readonly url_openqa='https://openqa.suse.de'
+readonly url_qam='https://qam.suse.de'
```

Comment:
https://github.com/os-autoinst/scripts/pull/326#discussion_r1673814934 still applies

Reply:
these are declared as global vars and used inside the functions as is, without any override with local vars

Reply:
oh geez, again wrong conversation, my bad. I meant https://github.com/os-autoinst/scripts/pull/326#discussion_r1673818872

Reply:
Just use variables with default assignments like in https://github.com/os-autoinst/scripts/blob/master/monitor-openqa_job#L15 so that anyone calling those scripts can override on the fly, e.g. `url_openqa=https://openqa.opensuse.org openqa-search-maintenance-core-jobs`

Reply:
Added default values